cpp/collections_module/algorithms/collections.hpp

```diff
@@ -0,0 +1,30 @@
+#include <mgp.hpp>
+
+
+void contains(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory){
+  mgp::memory=memory;
+  const auto arguments=mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try{
+    mgp::List list=arguments[0].ValueList();
+    mgp::Value value=arguments[1];
+    bool contains_value=false;
+    if(list.Size()==0 || list.Empty()){
+      contains_value=false;
```

Comment:
you can set value on record here and use "early return" concept, you can check it here: https://medium.com/swlh/return-early-pattern-3d18a41bba8